plugin/evm/validators/interfaces/noop_state.go

```diff
@@ -0,0 +1,63 @@
+package interfaces
```

Comment:
Seems a little odd to put this under the `interfaces` package when it is an implementation and doesn't have any new interface definitions